src/plugins/ioredis.test.js

```diff
@@ -69,7 +69,7 @@ test('Bails if timeline is not instance of performance-node', () => {
   expect(bool).toBe(false);
 });
 
-describe('Wrapping Redis Mock', () => {
+xdescribe('Wrapping Redis Mock', () => {
```

Comment:
looks like a typo?

Reply:
Great catch--that is a typo in that it's a shorthand for skipping/disabling a test.  (I disable the mocks locally so I can test against actual Redis, but those tests fail in Circle, which needs the mock.)